<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 16pt &quot;JasmineUPC&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
font: 75 12pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string>同意并注册账号</string>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_coder">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Terminal&quot;;</string>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_passwd2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>189</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Terminal&quot;;</string>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_passwd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Terminal&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重复输入密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>验证码：</string>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_accout">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Terminal&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images/picture.qrc">:/image/注册界面.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lab_acc">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 9pt &quot;Aharoni&quot;;
font: 75 11pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_pwd">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 9pt &quot;Aharoni&quot;;
font: 75 11pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_pwd2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 9pt &quot;Aharoni&quot;;
font: 75 11pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_coder">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 9pt &quot;Aharoni&quot;;
font: 75 11pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
   </property>
   <property name="text">
    <string>换一张</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 75 9pt &quot;Aharoni&quot;;
color: rgb(85, 0, 0);
font: 75 16pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string>天蝎直播平台账号注册</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="cLayout"/>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>（区分大小写）</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_Box">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>同意条款</string>
   </property>
  </widget>
  <zorder>lab_back</zorder>
  <zorder>btn_register</zorder>
  <zorder>le_coder</zorder>
  <zorder>le_passwd2</zorder>
  <zorder>le_passwd</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>le_accout</zorder>
  <zorder>label</zorder>
  <zorder>lab_acc</zorder>
  <zorder>lab_pwd</zorder>
  <zorder>lab_pwd2</zorder>
  <zorder>lab_coder</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_8</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>label_5</zorder>
  <zorder>ch_Box</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">MyLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images/picture.qrc"/>
 </resources>
 <connections/>
</ui>
